core/domain/topic_domain.py

```diff
@@ -528,14 +528,14 @@ class VersionedSubtopicsDict(TypedDict):
     """Dictionary that represents versioned subtopics."""
 
     schema_version: int
-    subtopics: List[Subtopic]
+    subtopics: List[SubtopicDict]
```

Comment:
Why is there a change here?

Reply:
Actually, it was done by mistake. The `subtopic` key is expecting a list of Subtopics dictionaries, not a Subtopic object. So, I corrected it.

Reply:
Ok, makes sense